packages/theming/apple-theme/src/appleTheme.macos.ts

```diff
@@ -46,6 +46,76 @@ export const appleComponents = {
       variant: 'bodyStandard',
     },
   },
+  RadioButton: {
+    tokens: {
+      borderColor: 'rgba(0, 0, 0, 0.15)',
```

Comment:
nit: separate to it's own variable since we re-use it or make it it's own token? There were some new tokens introduced in the Checkbox PR for an example of new tokens